pkg/logger/logger.go

```diff
@@ -12,6 +12,7 @@ const (
 	markPass    = "✅ "
 	markSuccess = "🎉 "
 	markFailed  = "❌ "
+	markWarning = "⚠️"
```

Comment:
Added a Warning method and emoji.

Reply:
This looks so small from my browser compared to the others. Is there a bigger more obvious one?

Reply:
We also don't need an icon, we have other places in the code where we just say "Warning: "

Reply:
Ha, yeah it displays weird for me too in my IDE. Appears to be a documented [issue](https://github.com/microsoft/vscode/issues/95502). In my terminal it shows correctly though. I'm not sure why it's showing incorrectly in some places.

I like its attention-grabbing properties in the terminal so I'd prefer to keep it. If you have strong feelings the other way though I can take it out.